tests/lib/leche-test.js

```diff
@@ -11,9 +11,7 @@
 // Requirements
 //------------------------------------------------------------------------------
 
-var sinon = compatRequire('sinon') || sinon,
```

Comment:
Since we removed the browser tests, we don't need this (it was causing a few issues).